sql/5_0_1-to-5_0_2_upgrade.sql

```diff
@@ -86,3 +86,503 @@
+  `ACT9CCNEAR` varchar (50) DEFAULT NULL,
+  `ACT10CCNEAR` varchar (50) DEFAULT NULL,
+  `ACT11CCNEAR` varchar (50) DEFAULT NULL,
+  `ACT5SCNEAR` varchar (50) DEFAULT NULL,
+  `ACT6SCNEAR` varchar (50) DEFAULT NULL,
+  `ACT7SCNEAR` varchar (50) DEFAULT NULL,
+  `ACT8SCNEAR` varchar (50) DEFAULT NULL,
+  `ACT9SCNEAR` varchar (50) DEFAULT NULL,
+  `ACT10SCNEAR` varchar (50) DEFAULT NULL,
+  `ACT11SCNEAR` varchar (50) DEFAULT NULL,
+  `ACT1CCNEAR` varchar (50) DEFAULT NULL,
+  `ACT2CCNEAR` varchar (50) DEFAULT NULL,
+  `ACT3CCNEAR` varchar (50) DEFAULT NULL,
+  `ACT4CCNEAR` varchar (50) DEFAULT NULL,
+  `MOTILITYNORMAL` char (3) NOT NULL DEFAULT 'on',
+  `MOTILITY_RS` char (1) DEFAULT '0',
+  `MOTILITY_RI` char (1) DEFAULT '0',
+  `MOTILITY_RR` char (1) DEFAULT '0',
+  `MOTILITY_RL` char (1) DEFAULT '0',
+  `MOTILITY_LS` char (1) DEFAULT '0',
+  `MOTILITY_LI` char (1) DEFAULT '0',
+  `MOTILITY_LR` char (1) DEFAULT '0',
+  `MOTILITY_LL` char (1) DEFAULT '0',
+  `MOTILITY_RRSO` int (1) DEFAULT NULL,
+  `MOTILITY_RLSO` int (1) DEFAULT NULL,
+  `MOTILITY_RRIO` int (1) DEFAULT NULL,
+  `MOTILITY_RLIO` int (1) DEFAULT NULL,
+  `MOTILITY_LRSO` int (1) DEFAULT NULL,
+  `MOTILITY_LLSO` int (1) DEFAULT NULL,
+  `MOTILITY_LRIO` int (1) DEFAULT NULL,
+  `MOTILITY_LLIO` int (1) DEFAULT NULL,
+  `NEURO_COMMENTS` text,
+  `STEREOPSIS` varchar (25) DEFAULT NULL,
+  `ODNPA` varchar (50) DEFAULT NULL,
+  `OSNPA` varchar (50) DEFAULT NULL,
+  `VERTFUSAMPS` varchar (50) DEFAULT NULL,
+  `DIVERGENCEAMPS` varchar (50) DEFAULT NULL,
+  `NPC` varchar (10) DEFAULT NULL,
+  `DACCDIST` varchar (20) DEFAULT NULL,
+  `DACCNEAR` varchar (20) DEFAULT NULL,
+  `CACCDIST` varchar (20) DEFAULT NULL,
+  `CACCNEAR` varchar (20) DEFAULT NULL,
+  `ODCOLOR` varchar (50) DEFAULT NULL,
+  `OSCOLOR` varchar (50) DEFAULT NULL,
+  `ODCOINS` varchar (50) DEFAULT NULL,
+  `OSCOINS` varchar (50) DEFAULT NULL,
+  `ODREDDESAT` varchar (20) DEFAULT NULL,
+  `OSREDDESAT` varchar (20) DEFAULT NULL,
+  PRIMARY KEY `neuro_link` (`id`),
+  UNIQUE KEY `id_pid` (`id`,`pid`)
+) ENGINE = InnoDB;
+
+CREATE  TABLE `form_eye_locking` (
+  `id`         bigint(20) NOT NULL COMMENT 'Links to forms.form_id',
+  `pid`        bigint(20)   DEFAULT NULL,
+  `IMP`        text,
+  `PLAN`       text,
+  `Resource`   varchar(50)         DEFAULT NULL,
+  `Technician` varchar(50)         DEFAULT NULL,
+  `LOCKED`     varchar(3)          DEFAULT NULL,
+  `LOCKEDDATE` timestamp  NOT NULL DEFAULT CURRENT_TIMESTAMP
+  ON UPDATE CURRENT_TIMESTAMP,
+  `LOCKEDBY`   varchar(50)         DEFAULT NULL,
+  PRIMARY KEY `locking_link` (`id`),
+  UNIQUE KEY `id_pid` (`id`,`pid`)
+) ENGINE = InnoDB;
+
+ALTER TABLE `form_eye_mag_orders`
+    CHANGE `id` `id` BIGINT(20) NOT NULL AUTO_INCREMENT,
+    CHANGE ORDER_PID to pid BIGINT(20) NOT NULL,
+    ADD `form_id` bigint(20) NOT NULL AFTER `id`,
+    DROP INDEX `VISIT_ID`;
+ALTER TABLE `form_eye_mag_orders`
+    ADD UNIQUE `VISIT_ID` (`pid`, `ORDER_DETAILS`, `ORDER_DATE_PLACED`);
```

Comment:
This should be in the original form_eye_mag_orders table creation above and not altered after.

Reply:
my mistake. I thought it was a new table. looks good then.